files/default/php/ci-modules.ini

```diff
@@ -0,0 +1 @@
+extension=mongo.so
```

Comment:
Why do you use a `cookbook_file` resource and not `template` for ini files ? I think it would be useful to provide attributes to customize these configuration files..